microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -318,87 +269,87 @@ Now that you've reviewed the `WebView` code, let's extend the functionality of o
 
    ![The current scene in Unity's Scene view](./hololens2-images/unity_completed_scene.png)
 
-1. Connect WebViewBrowser prefabs:
-   1. Select the WebViewBrowser.
-   1. Drag **Back Button** from the Hierarchy pane into the **Back Button** variable field for the WebView Browser in the Inspector.
-   1. Drag **Go Button** from the Hierarchy pane into the **Go Button** variable field for the WebView Browser in the Inspector:
+Now that we have our UI designed, we will now create the code to hook up the buttons.  To do that, we'll create a new script that derives from `Microsoft.MixedReality.WebView`.
```

Comment:
```suggestion

<!-- ------------------------------ -->
#### Hook up buttons

Now that we have our UI designed, we will now create the code to hook up the buttons.  To do that, we'll create a new script that derives from `Microsoft.MixedReality.WebView`.
```
the numbering restarts at 1 here.  it's a long list of steps.  need to break up by adding heading.
Check doc-design: gh rendered:
https://github.com/michaelfarnsworth/edge-developer/blob/user/RobertVS-MS/WV2-HoloLens-GettingStarted/microsoft-edge/webview2/get-started/hololens2.md#step-7---extending-webview2-functionality

Ideally, might need additional h4 subheadings somewhere.

Reply:
in the other PR, will add:
<!-- ------------------------------ -->
#### Add the input field and buttons

<!-- ------------------------------ -->
#### Add code to hook up the buttons